src/testing/fast_array_utils/_array_type.py

```diff
@@ -120,7 +123,9 @@ def cls(self) -> type[Arr]:  # noqa: PLR0911
                 import cupy as cp
 
                 return cast("type[Arr]", cp.ndarray)
-            case "cupyx.scipy.sparse", ("csr_matrix" | "csc_matrix") as cls_name, None:
+            case "cupyx.scipy.sparse", (
+                "csr_matrix" | "csc_matrix" | "coo_matrix"
+            ) as cls_name, None:
```

Comment:
Maybe I'm a little lost here - why do we need `coo_matrix` here? Doesn't this mean that `_to_scipy_sparse` now also will convert to this class?

Reply:
This means that `ArrayType("scipy.sparse", "coo_matrix")` works now – not completely and officially (e.g. not all types can be converted into it), but enough for the internal `coo_matrix_type` fixture that’s used to check if `to_dense` supports `coo_matrix`.

Reply:
I suppose that's fair since we don't do any validation on the string initiators for `ArrayType`